tshirts/TShirtsTest.java

```diff
@@ -0,0 +1,12 @@
+
+public class TShirtsTest {
+	public void testSize() {
+		assert (tshirts.size(37) == "S");
+		assert (tshirts.size(38) == "S");
```

Comment:
you expect size 38 to be both 'S' and 'M' ? that is not clear expectation from the production code.

Reply:
Resolved, Could you please validate now. Thankyou!!